cli/reporter/print_compact.go

```diff
@@ -242,6 +245,23 @@ func (r *defaultReporter) printAssetsByPlatform(assetsByPlatform map[string][]*p
 			} else {
 				assetScoreRating = policy.ScoreRating_error
 				assetScore = "X"
+				if r.data.Errors[assetsByPlatform[platform][i].Mrn] != nil {
+					es := r.data.Errors[assetsByPlatform[platform][i].Mrn]
+					ei := errdetails.ErrorInfo{}
+					pberr := anypb.UnmarshalTo(es.Details[0], &ei, proto.UnmarshalOptions{})
```

Comment:
The explorer ErrorStatusCode is buried in the details, so I have to unpack it here.